src/neptune/internal/types/stringify_value.py

```diff
@@ -39,6 +39,9 @@ class StringifyValue:
     def __init__(self, value: Any):
         self.__value = value
 
+    def __getitem__(self, item):
```

Comment:
The same applies here, you cannot assume that it's an array all the time :wink:.

Reply:
Ok, but shall I leave them in `tests/unit/neptune/new/test_stringify_unsupported.py`? There we have a type hint of `str` in the constructor, so those methods will make sense for an underlying value of type string.

Reply:
No, `Obj` should be a really simple value like `int` or `string` but not a built-in :wink:. `len` and index-based access looks like an array/list.